packages/web-components/fast-foundation/src/patterns/start-end.ts

```diff
@@ -59,15 +56,15 @@ export const endSlotTemplate: (
     context: ElementDefinitionContext,
     definition: EndOptions
 ) => html`
-    <span
+    <slot
```

Comment:
I realize this is potentially a large stylistic breaking change. I asked in Discord but didn't get any response about the history of this structure. The issue is that the `span` always *exists* and therefore doesn't work with `flex` layout and `gap` because there is always space between the `span`s even if the `slot` is empty. With only the `slot` if it's empty it's treated correctly in layout and the `gap` is not added.

This seems semantically preferable to me, and seems to work with the existing events and notifications we have setup.

Reply:
TLDR - one PR one change is preferable to avoid scenarios like this where an entire change is held up by something unrelated. If it _is_ related, we'll want to look and ensure *every* API continues to work as expected. I'm not sure that's the case unless we actually test for it. Making a break like this and assuming it works doesn't give me much confidence. Can we check that everything *does* work and then we can look at the implications of making a breaking change to a template used by most controls?